src/Symfony/Component/Intl/Resources/data/timezones/root.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Version": "2.1.49.82",
+    "Meta": {
```

Comment:
root metadata is now compiled into root.json, in order to de-duplicte locale metadata